src/itemview_formaction.cpp

```diff
@@ -211,7 +211,9 @@ void itemview_formaction::process_operation(operation op, bool automatic, std::v
 	case OP_OPENINBROWSER:
 		LOG(level::INFO, "view::run_itemview: starting browser");
 		v->set_status(_("Starting browser..."));
-		v->open_in_browser(item->link());
+		if (!v->open_in_browser(item->link())) {
+			v->show_error(_("Browser failed to open the link!"));
+		}
 		v->set_status("");
```

Comment:
The problem here wasn't fixed: if a link fails to open in a browser and an error is shown, `set_status` will delete the error message. Better put it into `else` branch.

This comment also applies to 4 similar places, below.